substratevm/src/com.oracle.svm.configure/src/com/oracle/svm/configure/config/SerializationConfiguration.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2020, 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2020, 2020, Alibaba Group Holding Limited. All Rights Reserved.
```

Comment:
Please fix all header files to use
`All rights reserved.` (after Alibaba Group Holding Limited)
So that it matches exactly `All rights reserved.` (lowercase `rights` and lowercase `reserved`